Sources/SwiftKuery/ConnectionPool.swift

```diff
@@ -23,8 +23,26 @@ import Dispatch
 
 // MARK: ConnectionPool
 
-/// A connection pool implementation.
-/// The pool is FIFO.
+/**
+ This class implements a First in First out connection pool. The pool maintains a cache of `ConnectionPoolConnection` instances, which can be reused for future requests, to enhance performance.
```

Comment:
please can we change "First in First out" to "first in, first out" - you can add (FIFO) in brackets after that text if you wish.